packages/example/src/components/account/AccountButton.tsx

```diff
@@ -2,12 +2,15 @@ import React, { useEffect, useState } from 'react'
 import { useEthers, shortenAddress } from '@usedapp/core'
 import { Button } from '../base/Button'
 import { Colors } from '../../global/styles'
+// @TODO replace once merged and published into @usedapp/core
```

Comment:
You can just use:
`import { useLookupAddress } from `@usedapp/core``

First you need to run:
'yarn build`
form main repo directory thou